doc/lua_api.txt

```diff
@@ -435,8 +422,7 @@ Example:
 #### `[opacity:<r>`
 Makes the base image transparent according to the given ratio.
 
-`r` must be between 0 and 255.
-0 means totally transparent. 255 means totally opaque.
+`r` must be between 0 (totally transparent) and 255 (totally opaque).
```

Comment:
I think it's already quite obvious that higher opacity values == lower transparency; Are the explanations totally necessary?

Reply:
It's not wrong to minimize the confusion which may arise due to the unclear parameter `r`. Only the `totally` words seem superfluous; otherwise fine.

Reply:
Agreed